android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -247,8 +251,15 @@ public void run() {
 
           JoinMeetingOptions opts = new JoinMeetingOptions();
           MeetingViewsOptions view = new MeetingViewsOptions();
-          if(paramMap.hasKey("participantID")) opts.participant_id = paramMap.getString("participantID");
           if(paramMap.hasKey("noAudio")) opts.no_audio = paramMap.getBoolean("noAudio");
+          /**
+              participant_id was removed from android options.
+              There is no propper documentations and it still exists in jave docs...
+              Maybe it was renamed to customer_key or so on. (todo check)
+              Waiting before further changes.
+           */
+//          if(paramMap.hasKey("participantID")) opts.participant_id = paramMap.getString("participantID");
```

Comment:
based on https://devforum.zoom.us/t/participant-id-generated-by-me-and-passed-in-zoommtg-join-is-changing-to-zoom-generated-id-websdk/18018/4 `participant_id` has been deprecated and will be removed in web sdk.
So I guess it applies to all sdks. The alternative is to use `email`

Reply:
it looks like it has been renamed to `customer_key` (https://marketplacefront.zoom.us/sdk/meeting/android/us/zoom/sdk/MeetingOptions.html#customer_key)